### PR TITLE
Signed extrinsics

### PR DESCRIPTION
**Pull request by Denis-Dinkov**

### CODE REVIEW

File: src/assets/styles/index.css [hunk 1]
```diff
@@ -144,3 +144,28 @@ html[data-color-scheme="dark"] #icon-checked {
 html[data-color-scheme="light"] #icon-checked {
   --svg-icon-fill: white;
 }
+
+.explorer-pages-table {
```
Comment (DimitarTAtanasov) [inline src/assets/styles/index.css:148]: Could we think of moving this or renaming it considering the place it is defined

File: src/views/signedExtrinsics/index.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  const [signedExtrinsics, setSignedExtrinsics] = useState<IMappedTransferExtrinsic[]>([]);
+  const [selectedExtrinsic, setSelectedExtrinsic] = useState<IMappedTransferExtrinsic | null>(null);
+  const refInitalExtrinsicsDisplayed = useRef(false);
+  const isLoading = blocksData.size === 0;
+
+  const filterExtrinsics = useCallback((extrinsics: IMappedBlockExtrinsic[] = []) => {
+    return extrinsics.filter(extrinsic => extrinsic.isSigned).reverse() as IMappedTransferExtrinsic[];
+  }, []);
+
+  const loadInitialData = useCallback(() => {
```
Comment (DimitarTAtanasov) [inline src/views/signedExtrinsics/index.tsx:41]: Maybe we could think of refactor the way we are getting the extrinsics list, as it is now being used on multiple places and iterating the whole structure in each component could eventually lead to lack of performance.

**Review by DimitarTAtanasov — CHANGES_REQUESTED** [review]

File: src/components/modals/modalShowJson/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      onClose={onClose}
+      className={cn(
+        'w-5/6',
+        'flex max-h-screen flex-col gap-8 overflow-auto p-6',
+        'border border-dev-purple-300',
+        'dark:border-dev-purple-700',
+        'transition-all duration-300 ease-in-out',
+      )}
+    >
+      <h5 className="self-start font-h5-bold">Extrinsics: ${extrinsic.id}</h5>
```
Comment (pivanov) [inline src/components/modals/modalShowJson/index.tsx:34]: do we need the $ sign here?

**Review by pivanov — COMMENTED** [review]
I think we can make this one better:
`src/components/chainDataList/index.tsx`
There are a lot of components with a lot of useEffects ... 

Also you can check branch `signed-extrinsics-small-improvements` to see some improvements related to modal

**Review by DimitarTAtanasov — CHANGES_REQUESTED** [review]

File: src/components/chainDataList/index.tsx
[the remark this answers is not in the document]
Comment (Denis-Dinkov) [reply src/components/chainDataList/index.tsx:1]: Absolutely, I will fix it in another PR because it involves a lot of changes.

**Review by DimitarTAtanasov — APPROVED** [review]

**Pull request merged**